inception_v3.py

```diff
@@ -146,7 +146,7 @@ def InceptionV3(include_top=True,
     input_shape = _obtain_input_shape(
         input_shape,
         default_size=299,
-        min_size=75,
+        min_size=32,
```

Comment:
Currently, this model will not train on either of the CIFARs because they consist of 32x32 images. Inception-v3 is designed to work with ImageNet images, which are generally 256x256. Since _some_ (but _not all_) conv layers in Inception-v3 have `padding='valid'` (e.g. see [here](https://github.com/eigenfoo/batch-renorm/blob/master/inception_v3.py#L167)), we are simply running out of dimensions. What to do?

Reply:
Lop off the latter parts of the network! Right now we only keep the first three mixed layers.